### ISSUE TITLE
setting new color list should reset ColorSet iterator

### ISSUE THREAD

**Issue by pravdinalex** [amcharts/amcharts4#944]
```
var colorSet = new am4core.ColorSet();
colorSet.next().hex // -> "#67b7dc" - from default scheme
colorSet.next().hex // -> "#6794dc" - from default scheme
colorSet.list = [
                am4core.color('#e60202'),
                am4core.color('#ab104f'),
                am4core.color('#a367dc'),
                am4core.color('#4b79b5'),
                am4core.color('#67b7dc'),
                am4core.color('#ffc107'),
                am4core.color('#ffeb3b')
            ];
colorSet.next().hex // -> "#a367dc" - third color from NEW list
```

I expect to get "#e60202" - the first color of new list.

**Comment by martynasma** [comment]
Totally agree. This will be implemented in the next version. Thanks for the suggestion!

In the interim, you may need to call `reset()` manually after setting data.

**Comment by martynasma** [comment]
Fixed in [4.1.5](https://github.com/amcharts/amcharts4/releases/tag/4.1.5).

## [4.1.5] - 2019-02-11

### Added
- New property `contextMenuDisabled` (default `false`) added to `Sprite`. If set to `true` it will prevent context menu (such as one displayed on right click) from displayed.

### Changed
- Setting `data` on `ColorSet` will now automatically reset iterator.

### Fixed
- `"rightclick"` event was essentially not working.
- Angular with router enabled were breaking charts in Safari and older Firefox.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
